src/loaders/zarrLoader.js

```diff
@@ -93,7 +93,11 @@ export default class ZarrLoader {
     const selection = [...this.chunkIndex];
     selection[this.xIndex] = null;
     selection[this.yIndex] = null;
+    selection[this.channelIndex] = null;
     const { data } = await source.getRaw(selection);
+    if (this.channelChunkSize > 1) {
+      return this._decodeChannels(data);
+    }
```

Comment:
Maybe add a comment that this loader is experimental, or moreover that the loaders within viv are experimental. Perhaps this could be included in the README. Both `tiff` and `zarr` are being better formalized within `vitessce-image-loader` and these at the moment just support the example data that we have.

Reply:
Ok.